plugins/modules/postgresql_db.py

```diff
@@ -312,8 +317,26 @@ def db_exists(cursor, db):
     return cursor.rowcount == 1
 
 
-def db_delete(cursor, db):
+def db_dropconns(cursor, db):
+    if cursor.connection.server_version >= 90200:
+        """ Drop DB connections in Postgres 9.2 and above """
+        query = """
+        SELECT pg_terminate_backend(pg_stat_activity.pid) FROM pg_stat_activity
+        WHERE pg_stat_activity.datname=%(db)s AND pid <> pg_backend_pid()
+        """
```

Comment:
```suggestion
        query = ("SELECT pg_terminate_backend(pg_stat_activity.pid) FROM pg_stat_activity "
                      "WHERE pg_stat_activity.datname=%(db)s AND pid <> pg_backend_pid()")
```
I'd suggest this format to avoid extra spaces. Same for the query below